packages/compass-crud/src/components/virtualized-document-json-view.tsx

```diff
@@ -109,7 +107,12 @@ const VirtualizedDocumentJsonView: React.FC<
       renderItem={renderItem}
       estimateItemInitialHeight={estimateDocumentInitialHeight}
       rowGap={spacing[200]}
-      className={className}
+      useStableScrollbarGutter={true}
+      paddingLeft={spacing[400]}
+      // paddingRight is only 6px because we are using stable scrollbar gutter
+      // which takes 10px already
```

Comment:
We might want to double check how it looks on windows, the scroll bars there work a bit differently from mac, probably not an issue, I have a mac if you'd like me to check (not a blocker we can do a follow up for any small style things).

Reply:
Tried it out - looks Good!